DatadogSessionReplay/Sources/Recorder/ViewTreeSnapshotProducer/ViewTreeSnapshot/NodeRecorders/UIImageViewRecorder.swift

```diff
@@ -7,6 +7,33 @@
 #if os(iOS)
 import UIKit
 
+@_spi(Internal)
+public struct UIImageResource: Resource {
+    let image: UIImage
+    let tintColor: UIColor?
+
+    init(image: UIImage, tintColor: UIColor?) {
+        self.image = image
+        self.tintColor = tintColor
+    }
+
+    public var identifier: String {
+        var identifier = image.srIdentifier
+        if let tintColorIdentifier = tintColor?.srIdentifier {
+            identifier += tintColorIdentifier
+        }
+        return identifier
+    }
```

Comment:
Because `image.srIdentifier` involves png compression and md5 calculation, it is very dangerous to expose this in public API. Even if it is hidden by `@_spi(Internal)`, how would we guarantee performance of this call in caller SDK (here RN SDK)?